docs/calculator.html

```diff
@@ -327,6 +327,103 @@ <h1><span class="badge bg-dark text-bg-dark" id="impact">low</span></h1>
             .querySelector("form")
             .addEventListener("reset", () => setTimeout(refreshScore, 0));
 
+        function populateFormFromURL() {
```

Comment:
what could work for `populateFormFromURL`, once the changes for `generateURLFromForm` are done, is something like this (untested):

```js
const searchParams = new URLSearchParams(window.location.search);
for (const [name, value] of searchParams) {
  document.querySelector(`[name="${name}"][value="${value}"]`).checked = true
}
```